rasa/cli/interactive.py

```diff
@@ -103,6 +103,7 @@ def check_training_data(args):
     story_directory, nlu_data_directory = data.get_core_nlu_directories(training_files)
     if not os.listdir(story_directory) or not os.listdir(nlu_data_directory):
         print_error(
-            "Cannot train initial Rasa model. Please provide NLU data and Core data."
+            "Cannot train initial Rasa model. Please provide NLU and Core data "
+            "using the '--data' argument."
```

Comment:
I guess you'll have to say "a directory of nlu and core data" right? Is there any way to pass separate files that aren't in a folder together?

Reply:
You can actually provide multiple files like `--data data/nlu.md data/stories.md new-data/stories2.md`. So should be fine.

Reply:
oh, awesome!